lib/hexpm_web/templates/docs/rebar3_private.html.md

```diff
@@ -0,0 +1,18 @@
+## Private packages and organizations
+
+Starting with rebar3 version 3.7.0, multiple Hex repositories (or indexes) can be used at the same time. Repositories are declared in an ordered list, from highest priority to lowest priority.
+
+When looking for a package, repositories are going to be traversed in order. As soon as one of the packages fits the description, it is downloaded. The hashes for each found packages are kept in the project's lockfile, so that if the order of repositories changes and some of them end up containing conflicting packages definitions for the same name and version pairs, only the expected one will be downloaded.
+
+This allows the same mechanism to be used for both mirrors, private repositories (as provided by hex.pm), and self-hosted indexes.
+
+### Publishing or using a private package
```

Comment:
I think this section only shows how to use a private package. What do I add to `app.src` to publish to a specific organization.

Reply:
Nothing changes in `.app.src`. But you do have to supply `-r <repo>` to `rebar3 hex publish` to specify the org name